include/aspect/particle/particle_handler.h

```diff
@@ -142,14 +158,22 @@ namespace aspect
         /**
          * Insert a particle into the collection of particles. Return an iterator
          * to the new position of the particle. This function involves a copy of
-         * the particle and its properties. Note that this function is of NlogN
+         * the particle and its properties. Note that this function is of O(NlogN)
          * complexity for N particles.
          */
         particle_iterator
         insert_particle(const Particle<dim,spacedim> &particle,
                         const typename parallel::distributed::Triangulation<dim>::active_cell_iterator &cell);
 
         /**
+         * Insert a number of particle into the collection of particles.
+         * This function involves a copy of the particles and their properties.
+         * Note that this function is of O(N) complexity.
```

Comment:
What's N here? The number of existing particles? Or the number of new particles?